lib/vbox/machine.go

```diff
@@ -115,221 +111,21 @@ func CheckMachine(machine, device string) error {
 }
 
 // Update virtualbox image
-func Update(typeFlag string) (err error) {
+func Update() error {
```

Comment:
__[gometalinter]__
* [gocyclo] cyclomatic complexity 18 of function Update() is high (> 15)

<!-- Comment by [SideCI](https://sideci.com) -->